src/OrchardCore.Modules/OrchardCore.ContentFields/Views/TextField-PredefinedList.Edit.cshtml

```diff
@@ -0,0 +1,48 @@
+@model OrchardCore.ContentFields.ViewModels.EditTextFieldViewModel
+@using OrchardCore.ContentFields.Settings
+@using OrchardCore.ContentManagement.Metadata.Models
+
+@{
+    var settings = Model.PartFieldDefinition.Settings.ToObject<TextFieldSettings>();
+    var listSettings = Model.PartFieldDefinition.GetSettings<TextFieldPredefinedListEditorSettings>();
+
+    var options = new List<SelectListItem>();
+    var selectedValue = String.IsNullOrEmpty(Model.Text) ? listSettings.DefaultValue : Model.Text;
+    
+    foreach (var option in listSettings.Options)
+    {
+        var isSelected = option.Value == selectedValue
+            || (String.IsNullOrEmpty(option.Value) && String.IsNullOrEmpty(selectedValue));
+        
+        options.Add(new SelectListItem { Text = option.Name, Value = option.Value, Selected = isSelected });
```

Comment:
Maybe for Localization we could do this : 
options.Add(new SelectListItem { Text = T[option.Name].Value, Value = option.Value, Selected = isSelected });
Else that would be part of a admin UI that would help localize these values.